src/pages/home/report/ReportActionItemSingle.js

```diff
@@ -90,16 +93,17 @@ function ReportActionItemSingle(props) {
                 style={[styles.alignSelfStart, styles.mr3]}
                 onPressIn={ControlSelection.block}
                 onPressOut={ControlSelection.unblock}
+                disabled={isWorkspaceActor}
```

Comment:
Why are we disabling the avatar icon itself? Shouldn't we show the workspace details page similar to what happens when you click on header. Or is it part of different fix PR?

Reply:
@cristipaval

Reply:
@cristipaval  Gentle ping. I am actually waiting on this response. If we are not going to do the workspace page routing as part of this PR, i will continue testing with the current changes.

Reply:
I think you are right, I should make this one behave the same as the click on the header.

Reply:
Similar issue when we click on the workspace name as well in the report preview. It showing a infinite loader page, we should route it to the workspace details page as well. 


https://github.com/Expensify/App/assets/46707890/2dc2b898-a739-40b2-afe6-4745b080c809

Reply:
@cristipaval

Reply:
fixed.